internal/common/consts.go

```diff
@@ -50,3 +50,6 @@ const (
 	URLRawQuery       = "urlRawQuery"
 	SDKReservedPrefix = "ds-"
 )
+
+// ServiceVersion indicates the version of the device service itself, not the SDK - will be overwritten by build
+var ServiceVersion string = "0.0.0"
```

Comment:
Don't we also need SDK version here? Version endpoint should be returning both and if V1 is not, V2 will.
See here in App Func SDK:
https://github.com/edgexfoundry/app-functions-sdk-go/blob/83416a6554981b86374d6e4afcd9b4305524ea32/internal/constants.go#L36

Reply:
I noticed device-service is slightly different from app-configurable, which relies on one more `version.go` to pass version info to Bootstrap function. Maybe we leave this until v2 as well so that we can update Bootstrap function at the same time?